examples/example_plugin/example_plugin/jobs.py

```diff
@@ -122,10 +153,12 @@ def receive_job_button(self, obj):
 
 
 jobs = (
+    ExampleDryRunJob,
     ExampleJob,
     ExampleHiddenJob,
     ExampleLoggingJob,
     ExampleJobHookReceiver,
     ExampleSimpleJobButtonReceiver,
     ExampleComplexJobButtonReceiver,
 )
+register_jobs(*jobs)
```

Comment:
If `register_jobs` is now a requirement for apps to call to provide jobs, this needs to get documented in the plugin dev docs. But it seems like we still have `registry["plugin_jobs"]` for that purpose, so what is this adding?

Reply:
If I'm not mistaken the hope is to do away with the need for the registry but that requires refactoring the `class_path` logic in `BaseJob`. Reading the code to see if we can use `register_jobs` (or something equivilant) to update the registry(because as it stands the developer needs to do both the `jobs` list/tuple as well as call `register_jobs`. 

@gsnider2195 @jathanism how do we register Git jobs with Celery?

Reply:
Does the job need to be tracked by the plugin registry or is the act of registering a job under the plugin's python namespace good enough? The `nautobot.core.celery.register_job` method is how jobs get registered to celery and in order for jobs to be real celery tasks that registration needs to happen at import time as far as I can tell.